int32 a
int32 b
---
int64 result_add
int64 result_mult
